src/pointer-analysis/value_set.cpp

```diff
@@ -194,8 +194,22 @@ void value_sett::output(
         result=from_expr(ns, identifier, o);
       else
       {
-        result="<"+from_expr(ns, identifier, o)+", ";
-
+        if(o.id()==ID_dynamic_object)
+        {
+          dynamic_object_exprt::recencyt recency;
```

Comment:
Put the declaration on the first use unless there's a reason not to.
dynamic_object_exprt::recencyt recency = ...;